credsweeper/app.py

```diff
@@ -95,40 +105,43 @@ def __init__(self,
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
     @staticmethod
-    def _get_config_path(config_path: Optional[str], config_file: str) -> Path:
+    def _get_config_path(config_path: Optional[str]) -> Path:
         if config_path:
             return Path(config_path)
         else:
-            return APP_PATH / "secret" / config_file
+            return APP_PATH / "secret" / "config.json"
 
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
-    def _get_config_dict(self,
-                         config_path: Optional[str],
-                         api_validation: bool,
-                         use_filters: bool,
-                         find_by_ext: bool,
-                         depth: int,
-                         doc: bool,
-                         size_limit: Optional[str],
-                         exclude_lines: Optional[List[str]],
-                         exclude_values: Optional[List[str]],
-                         config_file: str = "config.json") -> Any:
-        config_dict = Util.json_load(self._get_config_path(config_path, config_file))
+    def _get_config_dict(
+            self,  #
+            config_path: Optional[str],  #
+            api_validation: bool,  #
+            use_filters: bool,  #
+            find_by_ext: bool,  #
+            depth: int,  #
+            doc: bool,  #
+            severity: Optional[Severity],  #
+            size_limit: Optional[str],  #
+            exclude_lines: Optional[List[str]],  #
+            exclude_values: Optional[List[str]]) -> Dict[str, Any]:
+        config_dict = Util.json_load(self._get_config_path(config_path))
         config_dict["validation"] = {}
         config_dict["validation"]["api_validation"] = api_validation
         config_dict["use_filters"] = use_filters
         config_dict["find_by_ext"] = find_by_ext
         config_dict["size_limit"] = size_limit
         config_dict["depth"] = depth
         config_dict["doc"] = doc
+        if severity:
+            config_dict["severity"] = severity.value
 
         if exclude_lines is not None:
             config_dict["exclude"]["lines"] = config_dict["exclude"].get("lines", []) + exclude_lines
         if exclude_values is not None:
             config_dict["exclude"]["values"] = config_dict["exclude"].get("values", []) + exclude_values
 
-        return config_dict
+        return config_dict  # type: ignore
```

Comment:
`type: ignore`? What does it mean?

Reply:
MYPY warns about returning Any. I've suggested this suppression in line will be better that full suppresion in config.